Sources/StreamChatUI/ChatMessageList/ChatMessage/ChatMessageContentView.swift

```diff
@@ -645,6 +669,29 @@ open class ChatMessageContentView: _View, ThemeProvider {
     @objc open func handleTapOnDeliveryStatusView() {
         delegate?.messageContentViewDidTapOnDeliveryStatusIndicator(indexPath?())
     }
+    
+    // MARK: - UITextViewDelegate
+    
+    open func textView(_ textView: UITextView, shouldInteractWith URL: URL, in characterRange: NSRange, interaction: UITextItemInteraction) -> Bool {
+        if let mentionedUsers = content?.mentionedUsers, !mentionedUsers.isEmpty {
+            let tappedMentionedUser = textViewUserMentionsHandler.mentionedUserTapped(
+                on: textView,
+                in: characterRange,
+                withMentionedUsers: mentionedUsers
+            )
+            tappedMentionedUser.map {
```

Comment:
Not sure what exactly happens here. Map is used for transforming arrays of elements, and here we're using it to call a delegate method. Could you provide more details why is this the case?

Reply:
Map here is being to unwrap the optional. But I agree this can be confusing to read, so I will change it 👍 The advantage is that it makes the code a bit more concise, but indeed not very readable

Reply:
Done ✅